src/Compilers/CSharp/Portable/Symbols/MultipleForwardedTypeSymbol.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed class MultipleForwardedTypeSymbol : ErrorTypeSymbol
```

Comment:
Do we have to add this type? Can we use MissingMetadataTypeSymbol.TopLevelWithCustomErrorInfo instead?